dockerfiles/che-fabric8/assets/branding/product.json

```diff
@@ -8,5 +8,8 @@
     "prefetch": {
       "cheCDN":"/api/cdn-support/paths"
     }
+  },
+  "header": {
+    "warning": "Eclipse Che hosted by Red Hat is going to be shut down on <strong>March 1, 2021 at 00:01 GMT</strong>. A new service based on <a href='https://developers.redhat.com/developer-sandbox#assembly-field-sections-59571'>CodeReady Workspaces</a> is available now. Details about the new service, including signing up for it, can be found <a href='https://developers.redhat.com/developer-sandbox'>here</a>."
```

Comment:
I don't think strong tag is supported
https://github.com/eclipse/che-dashboard/pull/152/files#diff-b54815a3469ad6b44f74516907393c946858d8a0f81a53003c4e893d7eae4246R32

It should be easy to add but nobody asked for that 
https://github.com/eclipse/che-dashboard/pull/152#issuecomment-772885788

Reply:
Maybe we should add everything from https://www.w3schools.com/html/html_formatting.asp

Reply:
right, no worries. this is not a problem.

Reply:
this is just cosmetics, so I removed the unsupported tag ;-)